src/main/resources/swagger/cromiam.yaml

```diff
@@ -730,6 +730,28 @@ paths:
             - openid
             - email
             - profile
+  '/engine/{version}/status':
+        get:
+          summary: Returns the current health status of any monitored subsystems
+          parameters:
+            - name: version
+              description: API Version
+              required: true
+              type: string
+              in: path
+              default: v1
+          tags:
+            - Engine
+          responses:
+            '200':
+              description: All subsystems report an "ok" status
+              schema:
+                $ref: '#/definitions/StatusResponse'
+            '500':
+              description: At least one subsystem does not have an "ok" status
+              schema:
+                $ref: '#/definitions/StatusResponse'
```

Comment:
Hmm can the caller disambiguate between "Cromwell responded to the status query but isn't in perfect health" and "Cromwell is too hosed to even process a status query"?

Reply:
This *is* the API, it's not created nor controlled by us

Reply:
grumble grumble seems like one would want to know the difference but if that's the standard whatevs

Reply:
BTW since you said Cromwell and not CaaS, note that if Cromwell is timing out that's what would show up here for Cromwell